libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ReleaseHelper.java

```diff
@@ -74,7 +114,11 @@ private void addFieldValueToRow(List<String> row, Release._Fields field, Release
                 addReleaseIdToRelationShipToRow(release.getReleaseIdToRelationship(), row);
                 break;
             case ATTACHMENTS:
-                row.add(release.attachments.size() + "");
+                String size = "0";
```

Comment:
You are right - I just did the minimal rework to keep it working and kept the implicit cast... Updated in new version.